exercise.schema.json

```diff
@@ -56,10 +56,12 @@
         },
         "correct_question_transition_id": {
           "description": "The question in this exercise to show when the user provides a correct answer. Only relevant for `conditional` type questions",
+          "type": ["number", "null"],
```

Comment:
@Graspleric I understand this is how we implemented this initially, but would it make more sense to migrate this (eventually) to correct_question_transition instead of the id? Since the correct_question_transition_id in the DB refers to the transition, not the specific question id right?
For me it would make sense to use the same approach for correct/incorrect transitions as for the answer transitions.

Reply:
@screambeard as discussed: I'll update this to be inline with the answer setup immediately.